files/system/river/usr/bin/startriver

```diff
@@ -5,11 +5,12 @@
 # builds actually ran successfully without any errors!
 set -oue pipefail
 
-
+wayblue_origin_conf="/usr/etc/river/init"
 river_conf="$HOME/.config/river/init"
 
-if [ -f "$river_conf" ]; then
-    river
-else
-    XDG_CONFIG_HOME=/usr/etc/ river
+# Replace the homedir conf if it's missing or of wayblue origin
+if [ ! -f "$river_conf" ] || grep -q "ORIGIN: WAYBLUE" "$river_conf"; then
```

Comment:
Use `-e` instead of `-f` here, more general test condition. I'm pretty sure `-f` would fail if `$river_conf` was a symlink but best to be safe.